packages/website/src/widgets/radius-box/radius-tool.tsx

```diff
@@ -0,0 +1,71 @@
+import { useRef, useState } from "react";
+import { Button, Input, PickList } from "@lucky-parking/ui";
+import { formatDecimals } from "./utils/formatDecimals";
+import { UnitConvertor, Units } from "./utils/unit-convertor";
+
+interface RadiusToolProps {
+  isRadiusBttnDisabled?: boolean;
+}
+
+const RadiusOptions = [
+  { value: "miles", text: "Miles" },
+  { value: "feet", text: "Feet" },
+  { value: "kiloMeter", text: "K" },
```

Comment:
Let's change `K` to `KM` and `kiloMeter` to `kilometer`, per [user story](https://github.com/hackforla/lucky-parking/issues/538).